src/test/scala/no/gdl/bookapi/controller/BooksControllerTest.scala

```diff
@@ -113,4 +113,32 @@ class BooksControllerTest extends UnitSuite with TestEnvironment with ScalatraFu
       chapter.title should equal (TestData.Api.Chapter1.title)
     }
   }
+
+  test("that GET /mine returns AccessDenied for an invalid user") {
+    get("/mine") {
+      status should equal (403)
```

Comment:
Perhaps assert that the reason is set as well?

Reply:
And maybe add another test to assert that an _invalid_ token yields the same behaviour, not only if a token is omitted?